source/schemes.ttl

```diff
@@ -865,3 +865,8 @@
     dc:alternative "UDK "@sv;
     rdfs:label "Universell decimalklassifikation"@sv .
 
+<aat> a :GenreFormScheme; # should we use ConceptScheme instead? aat as Scheme?
```

Comment:
Yes, it seems AAT contains concepts other than `:GenreForm`, so `:ConceptScheme` is more appropriate.